web-ui/src/app/components/scrollable-table/scrollable-table.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="is-scrollable">
```

Comment:
Couldn't this be done with some existing component like https://www.primefaces.org/primeng/#/table?